### DISCUSSION TITLE
Feature request: Make exiting the terminal also exit Zed.

### DISCUSSION THREAD

**Discussion by LuckyLuc96** [zed-industries/zed#36950] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Hello. I have a silly idea that I think would be really cool. I know that the terminal that runs with zed does not get closed when you minimize it to the bottom dock (see video below), which works out great for my proposed setting. 

I propose adding an optional setting which would make it so that typing exit in the terminal, which closes the terminal, will also will close Zed. I think it would be a neat way to shutdown and stop working. 

### If applicable, add mockups / screenshots to help present your vision of the feature

Here I demonstrate that the terminal still exists when minimized to the dock. I believe adding a setting to close the editor after the terminal truly exits would be a great QOL feature and would not be detrimental or require a large deviation from more important updates. 

[Screencast from 07-15-2024 10:07:26 AM.webm](https://github.com/user-attachments/assets/95b8f934-7418-439e-85da-cc48e8e9d16b)

Thank you for your time and consideration :)

**Comment by notpeter** [comment]
We've floated some ideas recently about how to make Zed a better standalone terminal and something like this would definitely be reasonable, perhaps as a command line argument or a settings value.  Thanks for reporting.
